diff --git a/docs/conf.py b/docs/conf.py
index d5111391..21f425cc 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -10,19 +10,8 @@ import os
     cwd=os.path.join(os.path.dirname(__file__), os.path.pardir),
 )
 
-# -- Project information -----------------------------------------------------
-
-github_url = 'https://github.com'
-github_sponsors_url = f'{github_url}/sponsors'
-
 # -- General configuration --
 
-extensions = [
-    'sphinx.ext.extlinks',  # allows to create custom roles easily
-    'jaraco.packaging.sphinx',
-    'rst.linker',
-]
-
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
@@ -36,11 +25,6 @@ exclude_trees = []
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
-# -- Options for extlinks extension ---------------------------------------
-extlinks = {
-    'user': (f'{github_sponsors_url}/%s', '@'),  # noqa: WPS323
-}
-
 # -- Options for HTML output --
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
